architecture/platforms/versal_acap.rst

```diff
@@ -0,0 +1,125 @@
+.. _versal_acap:
+
+#############################################
+VERSAL Adaptive Compute Acceleration Platform
```

Comment:
Jorge just a generic comment please define all acronyms before use and define ones that are already spelled out. e.g. Physically Unclonable Function (PUF) line 18 below.